pkg/config/validator.go

```diff
@@ -98,31 +107,121 @@ type (
 		kind Kind
 		name string
 	}
+
+	// Key is used to lookup a policy.
```

Comment:
do we need to define policy here?  or is it clear that this is a chunk of specific mixer config?